src/lib.rs

```diff
@@ -79,6 +79,8 @@ pub struct State<'a> {
     pub image_stack: Vec<ImageLink<'a>>,
     /// Keeps track of the last seen heading's id, classes, and attributes
     pub current_heading: Option<Heading<'a>>,
+    /// True whenever between `Start(TableCell)` and `End(TableCell)`
+    pub in_table_cell: bool,
```

Comment:
Let's also make this entire `struct` `#[non_exhaustive]` so that this is the last breaking change just because the state changes.
This is unrelated to the PR, just some maintenance on top.